source/patterns/@aws-solutions-constructs/aws-events-rule-sns/lib/index.ts

```diff
@@ -0,0 +1,83 @@
+/**
+ *  Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License"). You may not use this file except in compliance
+ *  with the License. A copy of the License is located at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  or in the 'license' file accompanying this file. This file is distributed on an 'AS IS' BASIS, WITHOUT WARRANTIES
+ *  OR CONDITIONS OF ANY KIND, express or implied. See the License for the specific language governing permissions
+ *  and limitations under the License.
+ */
+
+import * as sns from '@aws-cdk/aws-sns';
+import * as events from '@aws-cdk/aws-events';
+import * as defaults from '@aws-solutions-constructs/core';
+import { Construct } from '@aws-cdk/core';
+import { overrideProps } from '@aws-solutions-constructs/core';
+import { ArnPrincipal } from '@aws-cdk/aws-iam';
+
+export interface EventsRuleToSNSTopicProps {
+    /**
+     * User provided props to override the default props for the SNS Topic.
+     *
+     * @default - Default props are used
+     */
+    readonly snsTopicProps?: sns.TopicProps
```

Comment:
Use topicProps? instead of snsTopicProps?.. Check other -sns- patterns for reference

Reply:
Update the files to modify the name to topicProps.